clusters-rke1/main.tf

```diff
@@ -53,6 +53,7 @@ locals {
   cloud_cred_name          = "${local.rancher_subdomain}-aws-cloud-cred"
   node_template_name       = "${local.rancher_subdomain}-aws-nt"
   nt_depends_on            = var.create_node_reqs ? module.shared_node_template[0] : data.rancher2_node_template.existing_nt[0]
+  cluster_name             = "${local.rancher_subdomain}-${var.cluster_name}-${terraform.workspace}"
```

Comment:
do we need something like a UUID to prevent name collisions or is it a very unlikely scenario ?

Reply:
I get it, sounds good then.